cfgov/paying_for_college/models/pages.py

```diff
@@ -1,4 +1,56 @@
-# -*- coding: utf-8 -*-
-# from __future__ import unicode_literals
+from __future__ import unicode_literals
 
-# home for future wagtail page models
+from wagtail.wagtailadmin.edit_handlers import (
+    ObjectList, StreamFieldPanel, TabbedInterface
+)
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.fields import StreamField
+
+from paying_for_college.models.blocks import GuidedQuiz
+
+from v1.atomic_elements import molecules, organisms
+from v1.models import CFGOVPage, CFGOVPageManager
+
+
+class StudentResourcesPage(CFGOVPage):
+    """A page to serve subpages of paying-for-college."""
+    header = StreamField([
+        ('text_introduction', molecules.TextIntroduction()),
+        ('featured_content', organisms.FeaturedContent()),
+    ], blank=True)
+
+    content = StreamField([
+        ('full_width_text', organisms.FullWidthText()),
+        ('guided_quiz', GuidedQuiz()),
+        ('info_unit_group', organisms.InfoUnitGroup()),
+        ('expandable_group', organisms.ExpandableGroup()),
+        ('expandable', organisms.Expandable()),
+        ('well', organisms.Well()),
+        ('raw_html_block', blocks.RawHTMLBlock(
+            label='Raw HTML block'
+        )),
+    ], blank=True)
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        StreamFieldPanel('content'),
+    ]
+    # Tab handler interface
+    edit_handler = TabbedInterface([
+        ObjectList(content_panels, heading='General Content'),
+        ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
+        ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+    ])
+    objects = CFGOVPageManager()
+
+    def get_template(self, request):
+        for block in self.content:
+            block.value['situation_id'] = block.id
```

Comment:
What is this logic doing? I believe that StreamField blocks should already have a unique ID. Could you use [this trick](https://github.com/wagtail/wagtail/pull/3593#issuecomment-315027698) with `{% include_block %}` to leverage it instead of defining a new variable?

Reply:
That's a great suggestion @chosak -- I'll give that a try.

Reply:
Given the template behavior of ListBlock children, I'm not sure where we'll need to bring in the block ID, or whether we'll need it at all in the end. 

I would like to leave this variable in place for this PR, so that it's available as the front-end adjusts to the new model structures. My goal will be to ditch it or try to use the {% include_block %} method.